Google/google_module/google_drive_reports.py

```diff
@@ -22,6 +22,7 @@ class GoogleReportsConfig(BaseModel):
     intake_key: str
     frequency: int = 20
     chunk_size: int = 1000
+    intake_server: str = "https://intake.sekoia.io"
```

Comment:
For `GoogleReportsConfig`, could you use [DefaultConnectorConfiguration``](https://github.com/SEKOIA-IO/sekoia-automation-sdk/blob/main/sekoia_automation/connector/__init__.py#L24-L27) as base class and remove all duplicate properties (ie `intake_key`, `intake_server` and `chunk_size`)?